index.html

```diff
@@ -67,7 +66,7 @@
         <!-- Starting Primary-->
         <div class="Primary-Head-line">
           <h1 class="h2 text-color-1 no-margin-bottom">
-            Hey there, I'm Omer and
+            Hey there, I'm Omer.
```

Comment:
Good job working on the mobile version 😃, however, you need to fix some part

- [ ] Kindly, make sure you stuck to the design as much as possible (`margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=7V3y8z3F5uwwsjzJ-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍. Also make sure you added the go down `button` 👍 


> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/228518105-a4ab0ff0-8d8c-4b90-b320-5e4765e0651c.JPG)| |![Capture0](https://user-images.githubusercontent.com/57408419/228518224-b17551c7-417a-4954-aa14-168a0c46189b.JPG)| |